nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -714,6 +731,16 @@ in
             noCheck = true; # fsck fails on a r/o filesystem
           };
       });
+    boot.initrd.extraUnits."rw-store.service" = lib.mkIf cfg.writableStore (pkgs.writeText "rw-store.service" ''
```

Comment:
Can't we use our regular systemd unit generation for this? It is a trivial unit yet doesn't allow any composition (without adding a unit named `rw-store.service.d/overrides.conf` or such?).

Reply:
I thought about this, but I wasn't 100% sure if that would be too much. I'll look into it...